include/range/v3/detail/config.hpp

```diff
@@ -294,6 +294,10 @@
 
 #endif  // RANGES_CXX_INLINE_VARIABLES
 
+#ifdef RANGES_DOXYGEN_INVOKED
+#define RANGES_INLINE_VARIABLE(type, name) \
+    inline constexpr type name{};
+#else
```

Comment:
Ah, I just noticed that Doxyfile.in has a "CONCEPT_REQUIRES_IMPL" predefined...and then not used anywhere. :/ Will fix that, then.